tokio-util/src/context.rs

```diff
@@ -12,21 +12,21 @@ use std::{
     pin::Pin,
     task::{Context, Poll},
 };
-use tokio::runtime::Handle;
+use tokio::runtime::Runtime;
 
 pin_project! {
     /// `TokioContext` allows connecting a custom executor with the tokio runtime.
     ///
     /// It contains a `Handle` to the runtime. A handle to the runtime can be
     /// obtain by calling the `Runtime::handle()` method.
-    pub struct TokioContext<F> {
+    pub struct TokioContext<'a, F> {
         #[pin]
         inner: F,
-        handle: Handle,
+        handle: &'a Runtime,
```

Comment:
Hmm...unlike the prior version, this now has a borrow in it, so it's not `'static`. You'd have to write
```rust
let rt = ...;
some_other_future_executor::spawn(async move {
    rt.wrap(some_future).await
});
```
rather than
```rust
let rt = ...;
some_other_future_executor::spawn(rt.wrap(some_future));
```
which seems like a minor ergonomic speed bump in what i imagine is more or less the most common use case for this?

Maybe we should just clone it...what do you think?

Reply:
So `Runtime` doesn't implement clone anymore. But you can achieve the same thing if you `async move { rt.wrap(fut) }` and if you warp the `Runtime` with an `Arc` you can clone before moving.

Reply:
We can revisit this in the near future.

Reply:
yup,

> So `Runtime` doesn't implement clone anymore. But you can achieve the same thing if you `async move { rt.wrap(fut) }` and if you warp the `Runtime` with an `Arc` you can clone before moving.

yup, i missed Carl's review feedback while looking at this. seems good to me, now that the arc is factored out. :+1: